### PR TITLE
Update README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by skanda890**
- [X] **I have read the [contributing guidelines](https://github.com/marticliment/WingetUI/blob/main/CONTRIBUTING.md#coding), and I agree with the [Code of Conduct](https://github.com/marticliment/WingetUI/blob/main/CODE_OF_CONDUCT.md)**.
- [X] **Have you checked that there aren't other open [pull requests](https://github.com/marticliment/WingetUI/pulls) for the same changes?**
- [X] **Have you tested that the committed code can be executed without errors?**
- [X] **This PR is not composed of garbage changes used to farm GitHub activity to enter potential Crypto AirDrops.**
Any user suspected of farming GitHub activity with crypto purposes will get banned. Submitting broken code wastes the contributors' time, who have to spend their free time reviewing, fixing, and testing code that does not even compile breaks other features, or does not introduce any useful changes. Thank you for your understanding.
-----
Update README.md and CONTRIBUTING.md

-----

### CODE REVIEW

**Comment by Lockszmith-GH** [comment]
Not a maintainer, never contributed, but I have to say the language changes OP submitted are not beneficial in any way.
The language added seems more convoluted, and unnecessarily verbose.

File: README.md [hunk 1]
```diff
@@ -234,15 +234,15 @@ Microsoft has implemented a few checks for the software available on Winget to m
 <br><p align="center"><i>Check out the <a href="https://github.com/marticliment/UniGetUI/wiki">Wiki</a> for more information!</i></p>
 
 ## Command-line parameters:
-`--daemon`: Start UniGetUI without spawning a new window. UniGetUI will run minimized on the system tray. UniGetUI is called with this parameter when launched at startup. **Autostart UniGetUI in the notifications area** must be enabled for this parameter to work.<br>
+`--daemon`: Start UniGetUI without spawning a new window. UniGetUI will run as a minimized on the system tray. UniGetUI is called with this parameter when launched at startup. **Autostart UniGetUI in the notifications area** must be enabled for this parameter to work.<br>
```
Comment (marticliment) [inline README.md]: This is not gramatically correct

**Review by marticliment — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -234,15 +234,15 @@ Microsoft has implemented a few checks for the software available on Winget to m
...
 `--welcomewizard` (or simply `--welcome`): Show a window to choose which package managers to use.<br>
 `--updateapps`: Enable automatic installation of available updates.<br>
 Since 3.0.0: `--report-all-errors`: May help debug crashes<br>
 Since 3.1.0: `--uninstall-unigetui`: Attempts to unregister UniGetUI from the notification panel, and silently quit<br>
 Since 3.1.0: `--migrate-wingetui-to-unigetui`: Attempts to migrate WingetUI data folders and shortcuts to UniGetUI, and silently quits<br>
 
 ### Installer command-line parameters:
-The installer is inno-setup based, so it supports regular Inno Setup command-line parameters. Additionally, it also supports the following parameters:
+The installer is inno-setup-based and supports regular Inno Setup command-line parameters. Additionally, it also supports the following parameters:
```
Comment (Lockszmith-GH) [inline README.md]: remove the dash before `based`.

If the 'so it' is bothering someone, I would rephrase as:
> The installer is inno-setup based. It supports all Inno Setup command-line parameters as well as the following:

File: README.md [hunk 3]
```diff
@@ -211,7 +211,7 @@ If this doesn't work, consider asking for help at [Winget's project page](https:
 
 A: This is a known limitation of Winget. 
 
-See more details on the issue https://github.com/microsoft/winget-cli/issues/2603.<br>
+Please take a look at more details on the issue https://github.com/microsoft/winget-cli/issues/2603.<br>
```
Comment (Lockszmith-GH) [inline README.md]: Feels completely unnecessary to me as a reader. What is the reasoning for this suggestion?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -7,8 +7,6 @@ This repository **respects** people, despite their race, gender, religion, heigh
 
 ---------------------------------
 
-
-
 # Contributing guidelines:
```
Comment (Lockszmith-GH) [inline CONTRIBUTING.md]: probably the only valid suggestion for the PR.

File: README.md (hunk 1 above)
Comment (skanda890) [reply README.md]: True, then why did Grammarly correct it?

**Comment by skanda890** [comment]
Please convert this to draft.

**Comment by marticliment** [comment]
You should have the option to do it yourself

image

**Comment by skanda890** [comment]
> You should have the option to do it yourself
> 
> image
> 

I am not currently in the web version, I am in the mobile version where there is no such button.

File: CONTRIBUTING.md (hunk 4 above)
Comment (skanda890) [reply CONTRIBUTING.md]: Too bad, not.

File: README.md (hunk 1 above)
Comment (marticliment) [reply README.md]: Gramarly does not understand well technical language, do not trust it 100%

File: README.md (hunk 3 above)
Comment (skanda890) [reply README.md]: It is more pleasing.

Comment (Lockszmith-GH) [reply README.md]: You're relying on a tool and not on your skill. The tool you are using, might be able to identify gramatical issues, but does not provide good suggestions.

If you want to change the sentance, "Please take a look at more details..." is not a good English phrasing. It is cumbersome.

An alternative phrasing might be: "For more details, see this issue"

File: README.md (hunk 1 above)
Comment (skanda890) [reply README.md]: Yeah sure.

**Comment by skanda890** [comment]
@marticliment, what change have you requested?
image

**Comment by skanda890** [comment]
I have made some new changes, so please review them.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -20,16 +20,16 @@ Before reading: All of the rules below are guidelines, which means that they sho
 
 #### Issues:
  - Please use the BUG/ISSUE template
- - Please be clear when describing issues.
+ - Please be sure to be clear when describing issues.
```
Comment (Lockszmith-GH) [inline CONTRIBUTING.md]: unnecessary, cumbersome.

maybe try: "Please be as clear as possible when ..."

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -20,16 +20,16 @@ Before reading: All of the rules below are guidelines, which means that they sho
...
  - Please fill out the form and DO NOT send empty issues with the information on the title.
- - Please make sure to check for duplicates as said in the BUG/ISSUE template.
+ - Please check for duplicates as said in the BUG/ISSUE template.
```
Comment (Lockszmith-GH) [inline CONTRIBUTING.md]: Both versions are a bit 'off'. Maybe this:

> Please search for possible duplicates, and only post new content. (As stated in the BIG/ISSUE template)

**Review by Lockszmith-GH — COMMENTED** [review]
Overall much better, I've added a few comments about new additions.

**Review by marticliment — DISMISSED** [review]

**Review by marticliment — APPROVED** [review]
